Assets/Src/Scripts/UI/HeaderButtonEvents.cs

```diff
@@ -0,0 +1,72 @@
+// Copyright 2021-2022 The SeedV Lab.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.UI;
+
+
+
+public class HeaderButtonEvents : MonoBehaviour{
+    
+    [SerializeField] private Animator _animatorHeader;
+    [SerializeField] private Animator _animatorViewport;
+
+    private const string _animatorShow = "Show";
+    private const string _animatorHide = "Hide";
+
+    // Start is called before the first frame update
+    void Start(){
+       
+    }
+
+    // Update is called once per frame
+    void Update(){
+        
+    }
+
+    public void OnHideUI(){
```

Comment:
Method order per Google style:

If the methods Start() and Update() have no "public" modifier, they should be ordered after the public methods such as OnShowUI, onClose, etc.